http/http_client.hpp

```diff
@@ -635,10 +640,24 @@ class ConnectionInfo : public std::enable_shared_from_this<ConnectionInfo>
             } */
             boost::asio::ssl::context sslCtx{
                 boost::asio::ssl::context::tlsv13_client};
-            sslConn.emplace(conn, sslCtx);
+            sslConn.emplace(*newconn.get(), sslCtx);
             setCipherSuiteTLSext();
+            BMCWEB_LOG_ERROR << "created  ssl Conn" << sslConn.has_value();
```

Comment:
Add a space between the statement and value. Also please add the destination and port in the traces

Reply:
Ack

Reply:
Not done?